pkg/service/service.go

```diff
@@ -138,13 +169,17 @@ type Service struct {
 
 	healthServer  healthServer
 	monitorNotify monitorNotify
+	envoyCache    envoyCache
 
 	lbmap         LBMap
 	lastUpdatedTs atomic.Value
+
+	// key: namespace/name
+	l7lbSvcs map[string]*L7LBInfo
```

Comment:
https://github.com/cilium/cilium/pull/18894/#discussion_r814965845

I'm referring to this :) - 
```

type ServiceID struct {
    Name      string `json:"serviceName,omitempty"`
    Namespace string `json:"namespace,omitempty"`
}
```

Reply:
This is a `pkg/k8s` concept, and as of now `pkg/service` does not import `pkg/k8s`. `pkg/service` uses `ServiceID` from `pkg/loadbalancer`, which is defined as:
```
loadbalancer/loadbalancer.go-// ServiceID is the service's ID.
loadbalancer/loadbalancer.go:type ServiceID uint16
```

But I see that using a struct of strings would be more efficient than formatting strings for keys, so I'll add a new `ServiceName` type for this in `pkg/service`.

Reply:
Done :-)